### ISSUE TITLE
Certificates configuration template changed, but the documentation has not

### ISSUE THREAD

**Issue by juliamagan** [wazuh/wazuh-documentation#5793]
## Description

After the testing performed [here](https://github.com/wazuh/wazuh/issues/15523#issuecomment-1332021717), we can see that the names of the nodes in the configuration template have been changed, so the steps to be followed must also be changed.

- [Current steps](https://documentation-dev.wazuh.com/v4.4.0-alpha1/installation-guide/wazuh-indexer/installation-assistant.html#wazuh-indexer-nodes-installation):

```
bash wazuh-install.sh --wazuh-indexer node-1
```

- Expected steps:

```
bash wazuh-install.sh --wazuh-indexer indexer-1
```

This change must be performed for all nodes installed with the assistant. In addition, the template shown at the [beginning](https://documentation-dev.wazuh.com/v4.4.0-alpha1/installation-guide/wazuh-indexer/installation-assistant.html#initial-configuration) must be changed:

- Current template:

```yaml
nodes:
  # Wazuh indexer nodes
  indexer:
    - name: node-1
      ip: <indexer-node-ip>
    #- name: node-2
    #  ip: <indexer-node-ip>
    #- name: node-3
    #  ip: <indexer-node-ip>

  # Wazuh server nodes
  # If there is more than one Wazuh server
  # node, each one must have a node_type
  server:
    - name: wazuh-1
      ip: <wazuh-manager-ip>
    #  node_type: master
    #- name: wazuh-2
    #  ip: <wazuh-manager-ip>
    #  node_type: worker
    #- name: wazuh-3
    #  ip: <wazuh-manager-ip>
    #  node_type: worker

  # Wazuh dashboard nodes
  dashboard:
    - name: dashboard
      ip: <dashboard-node-ip>
```

- Expected template:

```yaml
nodes:
  # Wazuh indexer nodes
  indexer:
    - name: indexer-1
      ip: <indexer-node-ip>
    - name: indexer-2
      ip: <indexer-node-ip>
    - name: indexer-3
      ip: <indexer-node-ip>
  server:
    - name: server-1
      ip: <server-node-ip>
      node_type: master
    - name: server-2
      ip: <server-node-ip>
      node_type: worker
    - name: server-3
      ip: <server-node-ip>
      node_type: worker
  dashboard:
    - name: dashboard-1
      ip: <dashboard-node-ip>
    - name: dashboard-2
      ip: <dashboard-node-ip>
    - name: dashboard-3
      ip: <dashboard-node-ip>
```

**Comment by DFolchA** [comment]
Hello,

The template present in the DEV repository was outdated and has been updated.

Thank you for reporting this.

**Comment by s-ocando** [comment]
Due to a confusion, an incorrect `config.yml` was uploaded to packages-dev.  
The correct `config.yml` is the one in the wazuh-packages 4.4 branch https://github.com/wazuh/wazuh-packages/blob/4.4/documentation-templates/wazuh/config.yml which matches the one included in the documentation. 
Renaming component in the configuration template leads to changes in the Wazuh indexer configuration file and has a lot of impact on the devops repositories. This change will probably be included for 4.5.
